code/game/dna/genes/goon_disabilities.dm

```diff
@@ -377,7 +377,7 @@
 	if (istype(usr,/mob/living/carbon/human/))
 		var/mob/living/carbon/human/H = usr
 
-		H.visible_message("\red <b>[H.name]'s flesh melts right off! Holy shit!</b>")
+		H.visible_message("<span class='warning'><b>[H.name]'s flesh melts right off! Holy shit!</b></span>")
```

Comment:
Danger, although this is commented out code so it doesn't really matter. Same thing below.